Description: Use libjs-mathjax instead of online version.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: not-needed

--- a/doc/conf.py.in
+++ b/doc/conf.py.in
@@ -49,6 +49,7 @@ needs_sphinx = '1.1'
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 #mathjax_path="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"
+mathjax_path="mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"
 extensions = [
         'sphinx.ext.todo',
         'sphinx.ext.mathjax',
